ethers-solc/tests/project.rs

```diff
@@ -41,9 +41,7 @@ fn can_get_versioned_linkrefs() {
         .unwrap();
 
     let project = Project::builder().paths(paths).ephemeral().no_artifacts().build().unwrap();
-
-    let compiled = project.compile().unwrap();
-    assert!(!compiled.has_compiler_errors());
```

Comment:
(I Can't comment on the file)

`assert!(!compiled.has_compiler_errors());`
->
`compiled.assert_success();`